### ISSUE TITLE
Not Compatible With Apatch!

### ISSUE THREAD

**Issue by Lynxs16** [yzyhk904/hifi-maximizer-mod#14]
Not working on Apatch!
[Install_2024-03-30T07_08_02.146780.log](https://github.com/yzyhk904/hifi-maximizer-mod/files/14809612/Install_2024-03-30T07_08_02.146780.log)

**Comment by yzyhk904** [comment]
I wouldn't support incompatible Magisk variants. Please use Official Magisk!

If you like try [Compatible Magisk-mirroring](https://github.com/yzyhk904/compatible-magisk-mirroring). This may give you a help.

**Comment by Lynxs16** [comment]
Apatch is not a Magisk variant. its a totally different root method and i
did try this module and unfortunately this doesn't work.

**Comment by yzyhk904** [comment]
If you are right, this module is for Magisk and nothing to do with Apatch. Don't try to install this module on Apatch.

**Comment by yzyhk904** [comment]
I updated [Compatible Magiks-mirroring](https://github.com/Magisk-Modules-Alt-Repo/compatible-magisk-mirroring). This should work on APatch now, but you need some extra editing for Hifi maximizer module ZIP file (see README in details).

**Comment by Lynxs16** [comment]
Thanks, I'll check it out!

**Comment by Lynxs16** [comment]
[image: Screenshot_20240405-114151.jpg][image:
Screenshot_20240405-114203.jpg][image: Screenshot_20240405-114217.jpg]
It still doesn't work!
I am using Apatch Ver 10657 which is the latest and after flashing the
magisk mirroring it is still giving me same error. i have attached the
installation log for hifi maximizer module if you want to take a look at it
and let me know if you need help testing it.

**Comment by yzyhk904** [comment]
I wrote extra editing was needed! Read README!

**Comment by Lynxs16** [comment]
Sorry I didn't understood what I have to do the instructions are not clear
to me as I am a donut can you please simplify the instructions?

**Comment by yzyhk904** [comment]
Did you read README? It's very simple. Replace every "magisk --path" with "ksu-magisk --path" in the ZIP file. OR use official Magisk instead of APatch.

**Comment by Lynxs16** [comment]
yeah i did read README and i will try it

**Comment by Lynxs16** [comment]
Hey I did manage to get the modules installed and i installed hifi
maximizer usb sample rate unlocker and audio jitter silencer. The audio
jitter silencer didn't require any editing though. After flashing i tested
a lossless song on apple music and everything worked fine and when i tried
an high res lossless song it played fine for like 10-15 sec and then it
started stuttering like crazy. I thought some other module might be
interfering so I uninstalled all modules I thought could interfere but
nothing worked. It still stuttered. Also, for some reason the flashlight
option was completely disabled i couldn't turn on the flash light after I
flashed the hifi maximizer module.

For Additional info I use apple music without any wavelet or dolby
atmos sound enhancement sort of application and i use the Abigail pro
dongle dac with the kz edx pro iems paired
